### DISCUSSION TITLE
Pre-trained model - how are they created?

### DISCUSSION THREAD

**Discussion by etlweather** [coqui-ai/STT#2060] [Q&A] [answered]
Hello, 

The docs https://stt.readthedocs.io/en/latest/AUGMENTATION.html mention various augmentation methods. I was wondering if the pre-trained models for download were created using these methods or they are straight from Librispeech and Common Voice unmodified?

I get OK results on my audio when I have very good recording but it quickly degrade when I use real-life recordings of meetings / phone calls as the audio has noise (like AC) and other "quality" (like roomy sounding, not full tonality as not close enough to mic, etc.) to it.

**Comment by reuben** [comment]
@JRMeyer ping

**Comment by JRMeyer** [comment, marked as answer]
Hi @etlweather,

there was quite a bit of augmentation used in training the v1.0 English model. The kind of augmentation was a simple variation on SpecAugment [[1](https://ai.googleblog.com/2019/04/specaugment-new-data-augmentation.html)].

### First 50 Epochs [[1](https://dpaste.com/4U4R8VG2Y)]

```
  "augment": [
    "frequency_mask[p=0.9,n=2:5,size=2]",
    "time_mask[p=0.9,n=3:4,size=25:100,domain=spectrogram]"
  ]
```

### Second 50 Epochs [[2](https://dpaste.com/7S7932DFA)]

```
  "augment": [
    "frequency_mask[p=0.9,n=5:7,size=2]",
    "time_mask[p=0.9,n=4:5,size=100:125,domain=spectrogram]"
  ]
```

### Final 50 Epochs [[3](https://dpaste.com/DRMJCEP44)]

```
  "augment": [
    "frequency_mask[p=0.9,n=5:7,size=2]",
    "time_mask[p=0.9,n=4:5,size=100:125,domain=spectrogram]"
  ]
```
